beaker-vue/src/pages/ChatInterface.vue

```diff
@@ -0,0 +1,491 @@
+                                v-tooltip.right="{
+                                    value: `${statusLabel}: ${beakerSessionRef?.activeContext?.slug || ''}`, 
+                                    showDelay: 300
+                                }"
+                                :label="beakerSessionRef?.activeContext?.slug"
+                                :loading="!(beakerSessionRef?.activeContext?.slug)"
+                            >
+                                <i class="pi pi-circle-fill" :style="`font-size: inherit; color: var(--${connectionColor});`" />
+                            </Button>
+                            <ResetButton :on-reset-callback="() => setContext({})"/>
+                            <Button
+                                @click="toggleFileMenu"
+                                v-tooltip.right="{value: 'Show file menu', showDelay: 300}"
+                                icon="pi pi-file-export"
+                                size="small"
+                                severity="info"
+                                text
+                            />
+                            <OverlayPanel ref="isFileMenuOpen" style="overflow-y: auto; height:40em;">
+                                <BeakerFilePane/>
+                            </OverlayPanel>
+                            <DarkModeButton :toggle-dark-mode="toggleDarkMode"/>
+                        </template>
+                        <template #center>
+                            <div class="vertical-toolbar-divider" />
+                        </template>
+                        <template #end>
+                            <a  
+                                href="/index" 
+                                v-tooltip.right="{value: 'To Notebook View', showDelay: 300}"
+                            >
+                                <Button
+                                    icon="pi pi-book"
+                                    size="small"
+                                    severity="info"
+                                    text
+                                />
+                            </a>
+                        </template>
+                    </VerticalToolbar>
+                </header>
+                <main style="display: flex; overflow-y: auto; overflow-x: hidden;">
+                    <div class="central-panel">
+                        <BeakerNotebook
+                            ref="beakerNotebookRef"
+                            :cell-map="cellComponentMapping"
+                            v-keybindings="notebookKeyBindings"
+                            :noEmptyStartingCell="true"
+                        >
+                            <ChatPanel
+                                ref="chatPanelRef"
+                                :selected-cell="beakerNotebookRef?.selectedCellId"
+                                v-autoscroll
+                            >
```

Comment:
I'd suggest instead of `help you do {stuff}` -> `help you {stuff}` since the verb might be specific to the `stuff` involved. Otherwise looks great. Will this be configurable? For Biome we'd want it to be called the `Biome Agent` actually so we don't cause confusion.

Reply:
The way I was imagining it, the entire text would be be able to be replaced for other applications such as Biome.